public/examples/upload/uploadApp.js

```diff
@@ -345,7 +345,11 @@
         }
         var baseUrl = (version === 'IPv6') ? 'http://' + testPlan.baseUrlIPv6 : 'http://' + testPlan.baseUrlIPv4;
 
-        var uploadHttpConcurrentTestSuite = new window.uploadHttpConcurrentProgress(baseUrl + '/upload', 'POST', 1, 15000, 15000, 2, uploadHttpOnComplete, uploadHttpOnProgress,
+        var concurrentRuns = 1;
+        if(uploadSize>= 10671896){
+          concurrentRuns=6
```

Comment:
Can we have the value of `6` be tracked by a variable (or const/config value)? The same in public/speed-testJS.js